src/util/Options.js

```diff
@@ -251,4 +269,19 @@ class Options extends null {
   }
 }
 
+/**
+ * The default settings passed to {@link Options.sweepers} (for v14).
+ * The sweepers that this changes are:
+ * * `threads` - Sweep archived threads every hour, removing those archived more than 4 hours ago
+ * <info>If you want to keep default behavior and add on top of it you can use this object and add on to it, e.g.
+ * `sweepers: { ...Options.defaultSweeperSettings, messages: { interval: 300, lifetime: 600 } })`</info>
+ * @type {SweeperOptions}
+ */
+Options.defaultSweeperSettings = {
```

Comment:
Shouldn't you also place these defaults in `createDefault`?

Reply:
No, I cannot remove the existing make cache sweeper options in semver minor and adding sweepers on top of it would cause a degree of performance degradation with no actual changes.

Reply:
You can add a TODO for later, unless you plan on making the semver major PR quickly after this is merged.

Reply:
That PR is the very next PR I am going to write, there's a lot more TODO, and this is high on the priority list in that TODO list.